naslib/optimizers/core/metaclasses.py

```diff
@@ -121,3 +121,9 @@ def get_checkpointables(self):
             (dict): with name as key and object as value. e.g. graph, arch weights, optimizers, ...
         """
         pass
+
+    def set_checkpointables(self, architectural_weights):
+        """
+        would set the objects saved in the checkpoint during last phase of training
```

Comment:
Since other functions also include this, maybe add a description of parameters and return types.

Reply:
+1, agreed.

Reply:
The type of the Args has been specified. This function has no return value.